### DISCUSSION TITLE
Two Device Groups For Split Color/White Lights

### DISCUSSION THREAD

**Discussion by zachcheatham** [arendst/Tasmota#19083] [Q&A]
Hi all, I have two ceiling fixtures that have a ccw light aimed downwards and an rgb strip aimed up. I've set `SetOption37 128` so that the white channels are split into a second light. I'm trying to configure device groups for both of these so that two separate Martin Jerry button type dimmers can act as power and dimmer remotes for them (one switch controls both white lights and the other controls both rgb strips.) Setting up two device groups and then configuring `DevGroupTie` on the fixtures works well for controlling power of the split lights separately, but brightness only works for the RGB strip on Dimmer1. Unless I'm missing something, it seems like a limitation of splitting a light and trying to use `DevGroupTie` on the second split light. Does anyone know of a workaround for this?

**Comment by sfromis** [comment]
As Tasmota only fully supports _one_ light, the special feature of `SetOption37` is not integrated all around, indeed not as part of Device  Groups, you'd have to synchronize the behavior outside of Device Groups. I've got similar lamps (Yeelight) using ESP32, and if I wanted such synchronization, I'd probably do some coding in Berry.
